.github/workflows/e2e-nvidia-t4-x1.yml

```diff
@@ -3,12 +3,18 @@
 name: E2E (NVIDIA Tesla T4 x1)
 
 on:
-  workflow_dispatch:
-    inputs:
-      pr_or_branch:
-        description: 'pull request number or branch name'
-        required: true
-        default: 'main'
```

Comment:
The regular `e2e.yml` job has workflow dispatch still - we could probably remove this from that job as well?

Reply:
yes, we can remove that now.